qa/L0_metrics/metrics_config_test.py

```diff
@@ -97,6 +98,17 @@ def test_cache_counters_missing(self):
         for metric in CACHE_COUNTER_PATTERNS:
             self.assertNotIn(metric, metrics)
 
+    # Histograms
+    def test_inf_histograms_decoupled_exist(self):
+        metrics = self._get_metrics()
+        for metric in INF_HISTOGRAM_DECOUPLED_PATTERNS:
+            self.assertIn(metric, metrics)
+
+    def test_inf_histograms_decoupled_missing(self):
```

Comment:
```suggestion
    # "Time to First Response" metric currently only exists for decoupled models
    # so verify that it is not reported for a non-decoupled model.
    def test_inf_histograms_nondecoupled_missing(self):
```

Reply:
histograms_decoupled means the set of histograms existing in decoupled models. It's missing in either non-decoupled model or in decoupled model but `--metrics-config histogram_latencies=true` is not set.